tests/unit/cli/vyper_compile/test_compile_files.py

```diff
@@ -72,12 +75,12 @@ def bar() -> FooStruct:
 
 
 @pytest.mark.parametrize("import_stmt,alias", SAME_FOLDER_IMPORT_STMT)
-def test_import_same_folder(import_stmt, alias, tmp_path, make_file):
+def test_import_same_folder(import_stmt, alias, chdir_tmp_path, make_file):
     foo = "contracts/foo.vy"
     make_file("contracts/foo.vy", CONTRACT_CODE.format(import_stmt=import_stmt, alias=alias))
     make_file("contracts/IFoo.vyi", INTERFACE_CODE)
 
-    assert compile_files([foo], ["combined_json"], paths=[tmp_path])
+    assert compile_files([foo], ["combined_json"])
```

Comment:
Can we test something a bit more specific than truthiness here? This is only testing that no exceptions are raised given it returns a dict. Maybe move lines 318-320 to a separate function and use it here

Reply:
hm, that's an old assertion. i'll just remove it

Reply:
actually i changed it to `not None`, in case compile_files ever decides to return `None` it will be caught by tests